packages/sdk-node/src/nodeTransport.ts

```diff
@@ -194,7 +194,7 @@ export class NodeTransport extends BaseTransport {
       }
     } else {
       try {
-        result = Buffer.from(result).toString('binary')
+        result = Buffer.from(result || '').toString('binary')
```

Comment:
I generally prefer the `??` nullish coalescing operator to the boolean union `||` to avoid `OR`ing `0` or `NaN` into the empty string.

Unlike `||` which returns its right-hand-side if the left-hand-side is any falsy value (JavaScript has eight falsy values: `0`, `-0`, `NaN`, `false`, `null`, `undefined`, the empty string, and `document.all`) `??` only returns the right-hand side in the event that the left-hand side is `null` or `undefined` ("nullish").